examples/nfs-volume-clone/README.md

```diff
@@ -0,0 +1,156 @@
+# Introduction
+
+This is an end-to-end example of using filesystem 'volume clone' as the foundation
+for PVCs, allowing clients to use the same base filesystem for
+different volumes.
+
+Clones on an Oracle ZFS Storage Appliance start from a snapshot of the
+share, the snapshot is then cloned and made available. A snapshot takes
+no space as long as the origin filesystem is unchanged.
+
+More information about snapshots and clones on the Oracle ZFS Storage
+Appliance can be found in the
+[Oracle® ZFS Storage Appliance Administration Guide,](https://docs.oracle.com/cd/F13758_01/html/F13769/gprif.html)
```

Comment:
FYI. It seems this is the external address: https://docs.oracle.com/en/storage/zfs-storage/zfs-appliance/os8-8-x/admin-guide/index.html, although the address above works too.